rca/enquire_to_study/forms.py

```diff
@@ -45,6 +45,13 @@ class EnquireToStudyForm(forms.Form):
         queryset=EnquiryReason.objects.all(), widget=forms.RadioSelect, empty_label=None
     )
 
+    enquiry_questions = forms.CharField(
+        label="Your questions (optional)",
+        help_text="If you have a specific enquiry or question, please include it here.",
+        required=False,
+        widget=forms.Textarea,
+    )
+
```

Comment:
Does this need adding to the export? https://github.com/torchbox/rca-wagtail-2019/blob/master/rca/enquire_to_study/wagtail_hooks.py#L34

Reply:
Ah you're right I missed that one - I've updated this now. Thank you!